scheduler/src/cook/mesos/api.clj

```diff
@@ -199,9 +199,37 @@
             (s/optional-key :env) {s/Keyword s/Str}
             (s/optional-key :labels) {s/Keyword s/Str}
             (s/optional-key :max-runtime) PosInt
+            (s/optional-key :before-commands) [CommandRequest]
+            (s/optional-key :after-commands) [CommandRequest]
             :cpus PosNum
             :mem PosNum})))
 
+(def JobUuids
+  "Schema for one or more job uuids in params"
+  {:job [s/Uuid]})
+
+(def Instance
```

Comment:
Do we have any sense on how these additions affect scheduler performance/query times?